### ISSUE TITLE
how to: portfolio slider

### ISSUE THREAD

**Issue by vivkay** [vivkay/pointer#4]
Hi @thomasjbradley, 

I'd like to learn how to make a sliding portfolio viewer like our grad Sam Nickerson did here: http://samantha-nickerson.ca/ 

I inspected her code and it looks like a full combo of HTML, CSS and JS and I'm not confident I can seamlessly replicate it (though I will try). Can you recommend which of your videos are best for figuring out how to do this?

THANKS SO MUCH!!

V

**Comment by vivkay** [comment]
Actually I'd love to learn how they did this: 

http://codyhouse.co/gem/hero-slider/ 

I downloaded the free source files but it seems a little complicated...I'm not quite sure where to start. :/

**Comment by thomasjbradley** [comment]
This is the one Samantha is using:
http://tympanus.net/codrops/2013/02/26/full-width-image-slider/

I’d probably start by looking at their example files and copying and pasting the HTML and CSS bits into my website to get it to work. Their tutorial looks pretty good, read through it and just copy and paste the code they suggest.

---

Again not what you're asking: Why use a hero carousel? Are you trying to hide information? Lots of user studies have shown that very few people look past the first slide.

**Comment by vivkay** [comment]
I am hiding information actually! I want to use it for the other screenshots of my phone, but they're not important enough to actually be in the flow of the website. They're more of an extra 'if you're particularly interested in seeing more screens, you can' type of thing. :)

Will give the tutorial a go!

Thanks!!

**Comment by vivkay** [comment]
...@thomasjbradley, I tried using the code and they make it look so seamless! I plugged in all of the files and copied the links from the index.html file ...but it's totally broken. I feel hopeless as a coder if I can't even copy and paste some code! Bah! Am I missing something?

Also would you recommend typing all of the code out? It seems like a lot of js and looks really complicated and confusing. I'd like to make it work but I'm wondering if this is too difficult? What do you think?

:pineapple:

**Comment by vivkay** [comment]
@thomasjbradley I had another go, I read through the tutorial but there are files in the source files that aren't mentioned in their notes. I put these in but I keep getting this error about this piece of code they included in the bottom of the html file. 

```
    <script>
        $( function() {
            $( '#cbp-fwslider' ).cbpFWSlider();
        } );
    </script>
```

And all of my images just appear in a vertical row! I looked at the other parts and they all make sense. All the files are connected, the ul elements are logically linked to the css files. The javascript file is way beyond me, I don't even know where to start but I connected them...

So it has to be this part, right??? Ahhhhhh is all coding so hard all the time Thomasssss!? You probably would have made 200 sparkly websites in the time it took me to break one... :(

**Comment by thomasjbradley** [comment]
No, don’t type all the code out—unless you want to—it’s just extra work.
I don’t see the error message when I look in console; has it been synced?

Code is always hard, we all have major stumbling blocks—but when you figure it out it’s amazing.
